entry_types/scrolled/package/src/frontend/playerControls/ControlBar.js

```diff
@@ -0,0 +1,129 @@
+import React, {useState} from 'react';
+import classNames from 'classnames';
+
+import SubtitlesIcon from '../assets/images/playerControls/subtitles_24px.svg';
+import SettingsIcon from '../assets/images/playerControls/settings_24px.svg';
+
+import {PlayPauseButton} from './PlayPauseButton'
+import {ProgressIndicators} from './ProgressIndicators'
+import {TimeDisplay} from './TimeDisplay'
+import {ContextMenu} from './ContextMenu'
+
+import styles from './ControlBar.module.css';
+import styleTransparent from './styles/ControlBarTransparent.module.css';
+import styleWhite from './styles/ControlBarWhite.module.css';
+import styleBlack from './styles/ControlBarBlack.module.css';
+
+export function ControlBar(props) {
```

Comment:
Since this is the main export that is used in outside code, I'd rename this file to `frontend/PlayerControls/index.js`. This makes it clear, that all the other files in this directory are implementation details. The face that you can turn a file that is getting to large into a directory with an index file without changing imports is one of my favorite traits of the ES6 module system.